app/javascript/dashboard/routes/dashboard/settings/auditlogs/Index.vue

```diff
@@ -80,6 +80,13 @@ export default {
       },
     };
   },
+  activated() {
+    this.$store.dispatch('auditlogs/fetch', { page: 1 }).catch(error => {
+      const errorMessage =
+        error?.message || this.$t('AUDIT_LOGS.API.ERROR_MESSAGE');
+      this.showAlert(errorMessage);
+    });
```

Comment:
@vishnu-narayanan Can you create a method named `fetchAllAuditLogs` and use these code inside the method and use the method here.